service-news/src/main/kotlin/crawler/ArticlePreview.kt

```diff
@@ -26,21 +31,21 @@ data class ArticlePreview(
         )
     }
 
-    val id: String
+    val id: Id<Article>
     val source: HpiMediaSource
 
     init {
         val matcher = URL_SOURCE_REGEX.matcher(url.toString())
         if (!matcher.find()) error("Source not in URL")
         source = SOURCE_MAPPING[matcher.group(1)] ?: error("Unknown source from URL")
 
-        val dateString = publishedAt.format(SORTABLE_DATE_FORMAT)
+        val dateString = SORTABLE_DATE_FORMAT.format(publishedAt.atZone(ZoneId.systemDefault()))
```

Comment:
We don't want to use the time zone of our server, but `Europe/Berlin` as the location of the HPI.